### PR TITLE
Adding policy support and in sync with master

### PR DESCRIPTION
**Pull request by jamengual**

### CODE REVIEW

File: output.tf [hunk 1]
```diff
@@ -1,19 +1,19 @@
 output "key_arn" {
-  value       = aws_kms_key.default.arn
+  value       = join("", aws_kms_key.default.*.arn)
   description = "Key ARN"
 }
 
 output "key_id" {
-  value       = aws_kms_key.default.key_id
+  value       = join("", aws_kms_alias.default.*.arn)
```
Comment (aknysh) [inline output.tf]:
```suggestion
  value       = join("", aws_kms_key.default.*.id)
```

Comment (jamengual) [reply output.tf]: done

File: output.tf [hunk 2]
```diff
@@ -1,19 +1,19 @@
 output "key_arn" {
-  value       = aws_kms_key.default.arn
+  value       = join("", aws_kms_key.default.*.arn)
   description = "Key ARN"
 }
 
 output "key_id" {
-  value       = aws_kms_key.default.key_id
+  value       = join("", aws_kms_alias.default.*.id)
```
Comment (aknysh) [inline output.tf]:
```suggestion
  value       = join("", aws_kms_key.default.*.key_id)
```

Comment (jamengual) [reply output.tf]: sorry, is my dyslexia , should be good now

**Pull request merged**